yark/archiver/config.py

```diff
@@ -70,6 +74,11 @@ def submit(self) -> None:
             )
             self.skip_download = True
 
+        # If running under docker, override some logical values
+        if os.environ.get('DOCKER_CONTAINER') == '1':
```

Comment:
`"` is the general project styling right now

Reply:
Updated